homework.js

```diff
@@ -0,0 +1,164 @@
+//summation off 5 function
+const summation = function(sum){
```

Comment:
this code only works if the argument is 5

make this function work regardless of the argument passed for example 

summation(6) // should return 21
summation(7)// should return 28